cmd/cosign/cli/verify/verify.go

```diff
@@ -500,3 +501,20 @@ func keylessVerification(keyRef string, sk bool) bool {
 	}
 	return true
 }
+
+func keylessVerificationWithSCTEnabled(ignoreSCT bool, keyRef string, sk bool, certChain string) bool {
```

Comment:
Can you rename this to `shouldVerifySCT`? Clearer what its purpose is.

Reply:
Agree i will change the name of function to shouldVerifySCT(ignoreSCT bool, keyRef string, sk bool)